annotations/helm-annotations/src/main/resources/NOTES.txt

```diff
@@ -0,0 +1,43 @@
+# Dekorate Autogenerated Notes
+
+To access the Helm annotations or properties you just need to have the following dependency in your
+class path:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>helm-annotations</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+Compile the project using:
+
+    mvn clean install
+
+You can find the generated Helm artifacts under: `target/classes/META-INF/dekorate/helm/<chart name>/` that should look like:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+
+How can we use it?
+
+First, make sure you have logged into a cluster with Helm enabled (in OpenShift clusters, Helm is enabled by default).
+
+Then, generate the Helm artifacts and push the image into a container registry:
```

Comment:
Can you maybe mention that the following line will not only generate the Helm chart for the project but also build/push the container runtime image to a registry

Reply:
Updated with:
```
Then, run the following Maven command in order to generate the Helm artifacts and build/push the image into a container registry:
```